src/lib/components/navbar/Navbar.component.js

```diff
@@ -48,6 +54,43 @@ const NavbarMenu = styled.div`
   align-items: center;
 `;
 
+const NavbarTabs = styled.div`
+  flex: 1;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+const TabItems = styled.div`
+  box-sizing: border-box;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0 ${defaultTheme.padding.base};
+  ${props => {
+    const brandingTheme = mergeTheme(props.theme, defaultTheme);
+    return css`
+      color: ${brandingTheme.primary};
+      &:hover {
+        border-bottom: 2px solid ${brandingTheme.primary};
+        span {
+          padding-top: 2px;
+        }
+        cursor: pointer;
+      }
+    `;
+  }};
+  ${props =>
+    props.selected &&
+    css`
+      border-bottom: 2px solid ${mergeTheme(props.theme, defaultTheme).primary};
+      span {
+        padding-top: 2px;
+      }
+    `};
+`;
+
 const NavbarMenuItem = styled.div`
```

Comment:
I think we should give`.sc-dropdown {
    cursor: pointer;
  }` and add ```cursor: pointer;``` for button in NavbarMenuItem so that the user can know it's clickable.

Reply:
good idea but we should update the Dropdown component instead of overriding the style here. Let's create another ticket please